plugins/include/cvars.inc

```diff
@@ -93,56 +113,59 @@ native get_cvar_pointer(const cvar[]);
+ * public cvar_change_callback(pcvar, const old_value[], const new_value[])
  *
- *       pcvar         - Pointer to cvar that was changed
- *       old_value     - String containing the value of the cvar before it was changed
- *       new_value     - String containing the new value of the cvar
+ *   pcvar         - Pointer to cvar that was changed
+ *   old_value     - Buffer containing the previous value of the cvar
+ *   new_value     - Buffer containing the new value of the cvar
  *
- *       The return value is ignored
+ * The return value is ignored
  *
- * @param pcvar         Pointer to cvar
- * @param callback      Name of callback function
+ * @param pcvar     Pointer to cvar
+ * @param callback  Name of callback function
  *
- * @return              Callback handle that can be used with [disable|enable]_cvar_hook
- * @error               Invalid cvar pointer or invalid callback function
+ * @return          Callback handle that can be used with
+ *                  [disable|enable]_cvar_hook
+ * @error           If an invalid cvar pointer or callback function is provided,
+ *                  an error will be thrown.
  */
 native cvarhook:hook_cvar_change(pcvar, const callback[]);
 
 /**
- * Stops a cvar hook forward from triggering.
+ * Disables a cvar hook, stopping it from being called.
  *
- * @note  Use the handle returned by hook_cvar_change as the parameter here.
+ * @note Use the handle returned by hook_cvar_change as the parameter here.
  *
- * @param handle        Forward to stop
- * @error               Invalid hook handle
+ * @param handle    Forward to disable
+ * @error           If an invalid hook handle is provided, an error will be
+ *                  thrown.
  */
 native disable_cvar_hook(cvarhook:handle);
 
 /**
- * Starts a cvar hook forward back up.
+ * Enables a cvar hook, restoring it to being called.
  *
- * @note  Use the handle returned by hook_cvar_change as the parameter here.
+ * @note Use the handle returned by hook_cvar_change as the parameter here.
  *
- * @param handle        Forward to back up
- * @error               Invalid hook handle
+ * @param handle    Forward to enable
+ * @error           If an invalid hook handle is provided, an error will be
+ *                  thrown.
  */
 native enable_cvar_hook(cvarhook:handle);
 
 /**
- * Returns flags of a cvar. The cvar is accessed by name.
+ * Returns a flags of a cvar. The cvar is accessed by name.
  *
  * @note For a list of possible flags see the FCVAR_* constants in amxconst.inc
- * @note Accessing a Cvar by name requires this function to walk through the
- *       engine's cvar list every time, which can result in a considerable waste
- *       of processing time, especially if many cvars have been registered. For
- *       a vastly superior alternative look at the get_pcvar_flags function.
+ * @note Accessing a Cvar by name is slower than direct pointer access, which is
```

Comment:
Well, such cvars are now relying on the cache, so technically it's true the first time, but once it's cached, not really. Well, it's fine to leave as it is though, but could be okay to add some note about it maybe?